ArduPlane/quadplane.cpp

```diff
@@ -893,7 +944,8 @@ void QuadPlane::update_transition(void)
     /*
       see if we should provide some assistance
      */
-    if (have_airspeed && aspeed < assist_speed &&
+    if (have_airspeed &&
+        assistance_needed(aspeed) &&
```

Comment:
This function is dependent on having an airspeed sensor but the code within it checks for airspeed and also angle error which is not related to airspeed. I suggest seperating those so angle check works without an airspeed sensor